core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -550,6 +547,7 @@ abstract class AbstractControllerBrokerRequestBatch(config: KafkaConfig,
         brokerEpoch, deletePartitions, partitions)
     }
 
+    stateChangeLog.trace(s"Sending StopReplica request (version $stopReplicaRequestVersion) to brokers ${controllerContext.liveOrShuttingDownBrokerIds}")
```

Comment:
I was wondering if the `debug()` logs a couple of lines below are better suited for the stateChangeLogger